conans/client/rest/auth_manager.py

```diff
@@ -179,26 +179,20 @@ def authenticate(self, user, password):
         remote_url = self._remote.url
         prev_user = self._localdb.get_username(remote_url)
         prev_username = prev_user or "None (anonymous)"
-        if not user:
-            self._user_io.out.info("Current '%s' user: %s" % (self._remote.name, prev_username))
+        user = None if user.lower() == 'none' else user
```

Comment:
Maybe protect it with:

``` user = None if user is None or user.lower() == 'none' else user```

I'm getting the following error when user is not type in command line:

```
12:13 $ conan user -p test
ERROR: 'NoneType' object has no attribute 'lower'
```

Reply:
Right, fixed. In fact this was hiding some misbehavior of the ``conan user`` command.